clang/test/DebugInfo/KeyInstructions/flag.cpp

```diff
@@ -8,8 +8,14 @@
 // HELP-NOT: key-instructions
 
```

Comment:
Not an issue with this patch but, with no positive HELP: prefixes, this check passes vacuously.